docs/internationalization.rst

```diff
@@ -0,0 +1,51 @@
+.. _chapter-i18n:
```

Comment:
I _think_ this can be dropped.  I don't think we will be supporting internationalization.